src/DependencyInjection/Configuration.php

```diff
@@ -31,6 +31,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('settingsDirectory')
                     ->defaultValue(null)
                 ->end()
+                ->scalarNode('batchSize')
```

Comment:
Unless you're planning to reuse the batch size for other operations as well, I'd suggest naming this `reindexBatchSize` to better document its use in the name.

Reply:
It's also used in any `index()` call, like `$indexManager->index($threeThousandObjects)` will split the data into chuck of 500 objects.

Reply:
And it's used in `$indexManager->remove($threeThousandObjects)` as well. That's why I'd rather keep `batchSize`.